src/core/currency/wallet/currency-wallet-files.js

```diff
@@ -49,7 +49,8 @@ export type TransactionFile = {
     tag?: string
   }>,
   secret?: string,
-  swap?: EdgeTxSwap
+  swap?: EdgeTxSwap,
+  deviceDescription?: string
```

Comment:
Can you put this line right above `feeRateRequested`? That would be alphabetical.